utils/utils.go

```diff
@@ -262,7 +292,7 @@ func WatchCerts() {
 // loadCerts loads public keys from the keys directory into a slice that is used
 // authenticating a user.
 func loadCerts() {
-	tmpCertHolder := make([]ssh.PublicKey, 0)
+	tmpCertHolder := make(map[string][]string)
```

Comment:
This should still be publickey. We want to compare them byte by byte.

Reply:
This still works as expected, the key in Extensions["pubKey"] is string anyway! Also, we need something comparable if we want a map

Reply:
You can still do a reference of the publickey string to a `ssh.PublicKey` object!

Reply:
I actually like the use of the struct retaining the public key and options. Then we can preload the options into a `map[string]string` to allow them for easy access (and therefore we can implement more of them easily!)

Reply:
Good, let's make options as `map[string][]string` because you can specify multiple permitlistens. We will do a poor man's multimap with this. We can also put `comments` into the structure. 
Ad `ssh.PublicKey` OK I can store that but not sure if there is any real reason